yapapi/executor/__init__.py

```diff
@@ -907,29 +969,34 @@ def __init__(
 
     @property
     def driver(self) -> str:
+        """Return the payment driver used for this `Executor`'s engine."""
         return self._engine.driver
 
     @property
     def network(self) -> str:
+        """Return the payment network used for this `Executor`'s engine."""
```

Comment:
just a thought - is this still required here? (apart from backwards-compatibility) ? ... maybe we should add `@deprecated` here?

Reply:
It's still used in blender-deprecated.py, a version of our Blender example that uses the Executor interface.
We probably don't need to deprecate it separately from the whole Executor class.